common/religion/holy_sites/wc_holy_sites.txt

```diff
@@ -0,0 +1,1326 @@
+	}
+}
+
+##naaru
+valaar = {
+	county = c_valaar
+	
+	character_modifier = {
+		name = holy_site_valaar_effect_name
+		opinion_of_different_culture = 20
+		diplomacy_per_prestige_level = 1
+	}
+}
+amberweb = {
+	county = c_amberweb
+	
+	character_modifier = {
+		name = holy_site_amberweb_effect_name
+		intimidated_vassal_tax_contribution_mult = 0.1
+		dread_gain_mult = 0.2
+	}
+}
+ammen_vale = {
+	county = c_ammen_vale
+	
+	character_modifier = {
+		name = holy_site_ammen_vale_effect_name
+		learning_per_piety_level = 1
+		same_culture_opinion = 5
+	}
+}
+#lordaeron
+#sunwell_plateau
+
+###############
+# lizard_group
+###############
+### saurok_religion
+ihgaluk = {
+	county = c_ihgaluk
+	
+	character_modifier = {
+		name = holy_site_ihgaluk_effect_name
+		dread_gain_mult = 0.15
+		tyranny_gain_mult = -0.1
+	}
+}
+nalaksha = {
+	county = c_nalaksha
+	
+	character_modifier = {
+		name = holy_site_nalaksha_effect_name
+		positive_random_genetic_chance = 0.1
+		positive_inactive_inheritance_chance = 0.1
+	}
+}
+thok = {
+	county = c_thok
+	
+	character_modifier = {
+		name = holy_site_thok_effect_name
+		jungle_advantage = 5
+		prowess_per_piety_level = 1
+	}
+}
+salyin = {
+	county = c_salyin
+	
+	character_modifier = {
+		name = holy_site_salyin_effect_name
+		prowess = 1
+		raid_speed = 0.1
+	}
+}
+#zanchul
+
+##############
+# loa_group
+##############
+#3 Seals of G'Huun
+zanchul = {
+	county = c_zanchul
+	
+	character_modifier = {
+		name = holy_site_zanchul_effect_name
+		councillor_opinion = 10
+		different_faith_opinion = 5
+	}
+}
+nazwatha = {
+	county = c_nazwatha
+	
+	character_modifier = {
+		name = holy_site_nazwatha_effect_name
+		prowess_per_piety_level = 1
+		wetlands_advantage = 5
+	}
+}
+atulaman = {
+	county = c_atulaman
+	
+	character_modifier = {
+		name = holy_site_atulaman_effect_name
+		desert_advantage = 3
+		desert_mountains_advantage = 2
+		knight_effectiveness_mult = 0.05
+	}
+}
+
+### zandism
+#zanchul
+#nazwatha
+#atulaman
+necropolis = {
+	county = c_necropolis
+	
+	character_modifier = {
+		name = holy_site_necropolis_effect_name
+		stress_loss_mult = 0.2
+		stress_gain_mult = -0.2
+	}
+}
+#surohi
+
+### gurubism
+#necropolis
+zulgurub = {
+	county = c_zulgurub
+	
+	character_modifier = {
+		name = holy_site_zulgurub_effect_name
+		prowess = 4
+	}
+}
+echo_isle = {
+	county = c_echo_isle
+	
+	character_modifier = {
+		name = holy_site_echo_isle_effect_name
+		naval_movement_speed_mult = 0.2
+		same_culture_mercenary_hire_cost_mult = -0.3
+	}
+}
+zulkunda = {
+	county = c_zulkunda
+	
+	character_modifier = {
+		name = holy_site_zulkunda_effect_name
+		powerful_vassal_opinion = 10
+	}
+}
+surohi = {
+	county = c_surohi
+	
+	character_modifier = {
+		name = holy_site_surohi_effect_name
+		intrigue = 2
+		hostile_scheme_resistance_mult = 0.4
+	}
+}
+
+### drakkism
+gundrak = {
+	county = c_gundrak
+	
+	character_modifier = {
+		name = holy_site_gundrak_effect_name
+		church_holding_holding_build_gold_cost = -0.4
```

Comment:
Ok, but imo, that's A LOT

Reply:
Changed in ce946f28151879466b666481478893f1d8abeb9c.